website/content/docs/agent/template.mdx

```diff
@@ -194,37 +205,17 @@ using the certificates `validTo` field.
 This does not apply to certificates generated with `generate_lease: true`. If set
 Vault Agent template will apply the non-renewable, leased secret rules.
 
-## Auto Auth and Templating Example
+When Agent's auto-auth triggers a re-authentication, e.g. due to token expiry,
```

Comment:
We avoid using *e.g.* as much as possible. 

Perhaps say: When Agent's auto-auth triggers a re-authentication, due to a token expiry for example, it generates a new token for Agent's use.

Reply:
Updated in [eb64fb9](https://github.com/hashicorp/vault/pull/15195/commits/eb64fb945c55953276443c88fcc87f1a662f00a9)